sample-apps/reviewer/client.py

```
@@ -0,0 +1,412 @@
+                headers=self.headers,
+                timeout=self.timeout,
+            )
+
+            if response.status_code == 200:
+                logger.info(f"Updated label {label_id}: {status} - {reviewer_name}")
+                return True
+            else:
+                logger.warning(f"Failed to update label {label_id}: {response.status_code}")
+                return False
+
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)